### PR TITLE
fix(release-notes): reorder and reword

### PR DESCRIPTION
**Pull request by NathalieC**
* reorder items to fit the Confluence 7.9 page
* add some of the missing items (more to come)

### CODE REVIEW

File: md/release-notes.md [hunk 1]
```diff
@@ -18,93 +21,153 @@ The others were left as they were. The main updated libraries can be found [ther
...
+It is composed of a few custom widgets.
+Provided as a Resource in Bonita Portal (the old one is deprecated), it can be imported and edited in Bonita UI Designer.
+It offers new features:
+   - A "list of applications" icon: once clicked, it displays a modal window that lists the available Applications for the profiles of the loggued user, so the user can pick one and switch application.
+   - The name of the loggued user: once clicked, it displays a modal window with user's information, the language picker for the application and a logout button
+It is compliant with all browsers: Microsoft Edge, IE11, Mozilla Firefox and Google Chrome.
+
+### Getting to Adaptive Case Management
+Bonita Platform now embeds the underlying mechanism that allows the update of tasks statuses in the scenario of mixed diagrams (structures as well as unstructured).
+This event-handler is in charge of the task_activity variable computation and is included in the Studio and the Runtime bundle but is disabled by default.
```
Comment (pabloalonso) [inline md/release-notes.md]: it is not clear about what it does.
This event-handler will evaluate task variable value after any task execution on the case to define the status of the task, it is included in Studio and Runtime bundle but is disabled bt default.

File: md/release-notes.md [hunk 2]
```diff
@@ -18,93 +21,153 @@ The others were left as they were. The main updated libraries can be found [ther
...
+   - A "list of applications" icon: once clicked, it displays a modal window that lists the available Applications for the profiles of the loggued user, so the user can pick one and switch application.
+   - The name of the loggued user: once clicked, it displays a modal window with user's information, the language picker for the application and a logout button
+It is compliant with all browsers: Microsoft Edge, IE11, Mozilla Firefox and Google Chrome.
+
+### Getting to Adaptive Case Management
+Bonita Platform now embeds the underlying mechanism that allows the update of tasks statuses in the scenario of mixed diagrams (structures as well as unstructured).
+This event-handler is in charge of the task_activity variable computation and is included in the Studio and the Runtime bundle but is disabled by default. 
+To use it, it needs to be enabled in the studio Preferences or in the bundle configuration.
+(doc link to add here)
+Moreover, a new action is available on the provided Submit button in the UI Designer : "assign and submit task". This action is mandatory for ACM use-case where a task can be taken by anybody to be executed.
```
Comment (pabloalonso) [inline md/release-notes.md]: Does it have sense to explain this feature on ACM? it will be extensively used out of ACM

**Review by pabloalonso — CHANGES_REQUESTED** [review]
Sry, I did not submit the review

File: md/release-notes.md (hunk 1 above)
Comment (NathalieC) [reply md/release-notes.md]: will modify. thx.

File: md/release-notes.md (hunk 2 above)
Comment (NathalieC) [reply md/release-notes.md]: Will add it as a new value on its own. Changing 7.9 Confluence page as well. Thx

**Review by pabloalonso — APPROVED** [review]

**Pull request merged**